### DISCUSSION TITLE
Choice of antennas

### DISCUSSION THREAD

**Discussion by mvmn** [cariboulabs/cariboulite#154] [Q&A] [answered]
Which antennas do you use for your Caribou Lite?

I presume I can use something like 915MHz antenna from FPV drone for the sub 1GHz range - though I'm not completely sure. But for 6GHz I don't have any 6GHz antennas available, and I reckon 5.8GHz from a WiFi router may not work properly - because of the standing wave ratio (SWR) mismatch etc.

Does anyone have any advices on this?

**Comment by K7MDL2** [comment, marked as answer]
For receive only the SWR (representing the 50ohm impedance match) is of less concern.  A simple piece of wire will work (at least somewhat) stuck into the SMA connector.  At VHF, UHF and microwave frequencies, location and directionality/gain/beamwidth of an antenna can give you big gains or losses.  

In simplest terms for the highest frequencies here, think of line of sight.  If you have an unobstructed view to the signal source, most anything will work.  Distance and obstructions reduce the signal requiring more antenna to overcome the path loss. 

Experiment with any and all antennas you have with a known signal, including FM radio stations, Marine weather, WWV.   I regularly use my 130ft HF wire antenna to receive strong VHF signals including ham radio repeaters, public safety, VHF marine weather, and FM broadcast.    

For serious VHF/UHF I use tuned omni-directional loops, directional yagi antennas, and for the highest bands, 18 to 30inch dishes.  The signal may be direct or a reflection off mountains or structures, or even ice crystals in high elevation storm clouds.

When choosing an antenna consider what frequency and where the source is.  What do you want to hear at 5 or 6GHz? Therefore, do you need that kind of antenna?

Ther are very wideband directional and non-directional antennas like discones and log periodic dipole arrays (LPDA).  Antennas and signals can be vertical, horizontal, or circular  polarized as well.  Goes back to what do want to listen to.  Right tool for the right job.
